plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/conf/hydra/sweeper/ax.yaml

```diff
@@ -34,5 +34,5 @@ hydra:
       verbose_logging: false
       max_trials: ${hydra.ax.max_trials}
       # set random seed here to make Ax results reproducible
-      random_seed: null
```

Comment:
if this is for the text, make it effective only in the test.
no need to change the default config on behalf of the test.

Reply:
Should a seed be set by default as a good practice (irrespective of tests)?

Reply:
no. it should not be set by default to achieve better robustness.
You should set it when you want reproducibility (while debugging for example).